README.md

```diff
@@ -117,7 +117,7 @@ $ cat robots.html | pup 'th[scope="row"]'
 
 CSS selectors have a group of specifiers called ["pseudo classes"](
 https://developer.mozilla.org/en-US/docs/Web/CSS/Pseudo-classes)  which are pretty
-cool. pup implements a majority of the relevant ones them.
+cool. pup implements a majority of the relevant ones of them.
```

Comment:
```suggestion
cool. pup implements a majority of the relevant ones.
```
Grammatically, it makes more sense to _remove_ the final word instead, since "CSS selector specifiers" is already implied by the previous sentence.